specification/dfp/resource-manager/Microsoft.Dynamics365Fraudprotection/preview/2021-02-01-preview/dfp.json

```diff
@@ -0,0 +1,647 @@
+          "Get details of an instance": {
+            "$ref": "./examples/getInstance.json"
+          }
+        },
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_GetDetails",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "instanceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the instance. It must be a minimum of 3 characters, and a maximum of 63."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/DFPInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "description": "Provisions the specified DFP instance based on the configuration specified in the request.",
+        "x-ms-examples": {
+          "Create instance": {
+            "$ref": "./examples/createInstance.json"
+          }
+        },
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "instanceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the DFP instances. It must be a minimum of 3 characters, and a maximum of 63."
+          },
+          {
+            "name": "instanceParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DFPInstance"
+            },
+            "description": "Contains the information used to provision the DFP instance."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/DFPInstance"
+            }
+          },
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

you should also add a 202, and x-ms-long-running-operation annotation - 201 is only for async create, 202 needs to be returned for async update

Reply:
Added 202 code to put, patch and delete